ui.R

```diff
@@ -37,8 +44,7 @@ shinyUI(fluidPage(
         column(6, numericInput(
           "wage1_hourly", "Hourly wage ($):",
           min = 18, max = 100,
-          #TY24 minimum wage
-          value = 22.70, step = .5,
+          value = DEFAULT_HOURLY_WAGE, step = .5,

```

Comment:
rogue comma?

Reply:
Yeah, I think it was already there, another good catch thanks